cpg_workflows/stages/large_cohort.py

```diff
@@ -12,73 +14,110 @@
     get_workflow,
     stage,
 )
+from metamist.graphql import gql, query
 
-from .genotype import Genotype
+if TYPE_CHECKING:
+    from graphql import DocumentNode
 
+    from hailtop.batch.job import PythonJob
 
-@stage(required_stages=[Genotype])
+
+@stage(analysis_type="combiner", analysis_keys=["combiner"])
```

Comment:
```suggestion
@stage(analysis_type="combiner")
```

Relevant code [here](https://github.com/populationgenomics/production-pipelines/blob/d5bf2b8ae33b5274acf16915ebde1075302a2316/cpg_workflows/workflow.py#L562) - if your `expected_outputs` method returns a dict, `analysis_keys` tells it which entry in the dict to register as an output. If the return is a Path, `analysis_keys` is ignored. I don't think it would break, but this arg conflicts with the lack of dictionary return here.